### PR TITLE
Downsampling: add more tests for downsample action

### PR DESCRIPTION
**Pull request by weizijun**
Add some tests for downsample action

### CODE REVIEW

**Comment by pxsalehi** [comment]
@elasticmachine test this please

File: x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml [hunk 1]
```diff
@@ -181,3 +190,63 @@ setup:
...
+        index: normal-index
+        target_index: rollup-test
+        body:  >
+          {
+            "fixed_interval": "1h"
+          }
+
+
+---
+"Rollup no metric index":
```
Comment (csoulios) [inline x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml:216]: I think a `skip` section is needed here.

**Review by csoulios — CHANGES_REQUESTED** [review]
Thanks for submitting those tests. 
I have left a small comment.

File: x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml (hunk 1 above)
Comment (weizijun) [reply x-pack/plugin/rollup/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/rollup/10_basic.yml:216]: done

**Review by csoulios — APPROVED** [review]

**Comment by csoulios** [comment]
@elasticmachine test this please

**Pull request merged**